### PR TITLE
Kidneys are now 2

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Adds another kidney, so there are 2 kidneys.

## Why It's Good For The Game

2 kidneys are better than one, also the doc requests 2 kidneys.

## Changelog
:cl:
add: Added another kidney.
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
now this is realism

File: code/modules/organs/internal/internal_organ_processes.dm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 /mob/living/carbon/human/proc/process_internal_ograns() //Calls all of the internal organ processes
 	if(should_have_process(OP_EYES))	//Bad way to do this, should be reworked to call procs from a list that's generated by the species.has_process list + extra processes gotten from organs (carrion)
 		eye_process()
-	if(should_have_process(OP_KIDNEYS))
+	if(should_have_process(OP_KIDNEY_LEFT) && should_have_process(OP_KIDNEY_RIGHT))
```
Comment (TorinoFermic) [inline code/modules/organs/internal/internal_organ_processes.dm:10]: In real life, humans can go fine with one kidney

Comment (TorinoFermic) [reply code/modules/organs/internal/internal_organ_processes.dm:10]: Usage of || is fine there

Comment (TheShown911) [reply code/modules/organs/internal/internal_organ_processes.dm]: Figgsed :DDDD

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**